### ISSUE TITLE
'Task' object has no attribute 'run_in_thread'

### ISSUE THREAD

**Issue by Taurus95** [ProtonVPN/linux-app#31]
I add the new ProtonVPN repository on my Debian 10 and install the app but when I try to log in I receive this error on console:

```bash
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/protonvpn_gui/view/login.py", line 235, in on_clicked_login
    self.login_view_model.login_async(username, password)
  File "/usr/lib/python3/dist-packages/protonvpn_gui/view_model/login.py", line 37, in login_async
    process.start()
  File "/usr/lib/python3/dist-packages/protonvpn_gui/patterns/factory/background_process_factory.py", line 84, in start
    self.process.run_in_thread(to_run_in_thread)
AttributeError: 'Task' object has no attribute 'run_in_thread'
```

I install the last updates today. Before this no login working and no logs. 
System info: 
```
Linux name 4.19.0-17-amd64 #1 SMP Debian 4.19.194-3 (2021-07-18) x86_64 GNU/Linux
```

**Comment by calexandru2018** [comment]
Hey @Taurus95 

It would help if you'd post the versions.

**Comment by Taurus95** [comment]
Today I realize that I was using the CLI community version, so I uninstall it and try to use the protonvpn-cli but it doesn't work well too. The login was complete successfully, so now when I try to use the GUI the error change a bit but basically is the same:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/protonvpn_gui/main.py", line 249, in do_activate
    win = self.get_dashboard_window()
  File "/usr/lib/python3/dist-packages/protonvpn_gui/main.py", line 281, in get_dashboard_window
    view_model=dashboard_view_model
  File "/usr/lib/python3/dist-packages/protonvpn_gui/view/dashboard.py", line 207, in __init__
    self._preload_ui_resources()
  File "/usr/lib/python3/dist-packages/protonvpn_gui/view/dashboard.py", line 210, in _preload_ui_resources
    self.dashboard_view_model.on_startup_preload_resources_async()
  File "/usr/lib/python3/dist-packages/protonvpn_gui/view_model/dashboard.py", line 202, in on_startup_preload_resources_async
    process.start()
  File "/usr/lib/python3/dist-packages/protonvpn_gui/patterns/factory/background_process_factory.py", line 84, in start
    self.process.run_in_thread(to_run_in_thread)
AttributeError: 'Task' object has no attribute 'run_in_thread'

```
OS: Debian 10 buster, Kernel: x86_64 Linux 4.19.0-17-amd64
Python version:
```
Python 3.7.3
```
Protonvpn packages:
```
ii  protonvpn                                     1.0.0-3                                 all          ProtonVPN metapackage
ii  protonvpn-cli                                 3.7.2-2                                 all          ProtonVPN CLI (Python 3)
ii  protonvpn-gui                                 1.1.0-7                                 all          ProtonVPN GUI (Python 3)
ii  protonvpn-stable-release                      1.0.1-1                                 all          Package to install ProtonVPN GPG key and stable repo
ii  python3-proton-client                         0.5.1-3                                 all          Proton SRP (Python 3)
ii  python3-protonvpn-nm-lib                      3.3.2-1                                 all          ProtonVPN NM library (Python 3)
```
If you need something more, just ask.

**Comment by calexandru2018** [comment]
Hey @Taurus95 

This should be fixed with the latest update 1.1.1, could you please confirm that ?

**Comment by Taurus95** [comment]
@calexandru2018 It is fixed! Thank you!. Now without KS it works fine, but with KS on, fall in error. I think that is something similar to #27 so I will be looking in the others issues.
